src/chromium/interoperability-with-cpp/example-bindings.md

```diff
@@ -0,0 +1,37 @@
+# Example bindings
+
+cxx requires you to declare the whole C++/Rust boundary in one of your `.rs`
+files. For instance:
+
+```rust
+#[cxx::bridge]
+mod ffi {
+    extern "Rust" {
+        type MultiBuf;
+
+        fn next_chunk(buf: &mut MultiBuf) -> &[u8];
+    }
+
+    unsafe extern "C++" {
+        include!("cxx-demo/include/blobstore.h");
+
+        type BlobstoreClient;
+
+        fn new_blobstore_client() -> UniquePtr<BlobstoreClient>;
+        fn put(&self, parts: &mut MultiBuf) -> u64;
+    }
+}
+```
+
+<details>
+Point out:
+
+* Native support for C++'s `std::unique_ptr` in Rust
+* Native support for Rust slices in C++
+* Calls from C++ to Rust (in the top part)
+* Calls from Rust to C++, and C++ types (in the bottom part)
```

Comment:
nit: Not sure why "types" are mentioned in the 2nd bullet but not in the 1st bullet (you have both an opaque Rust type *and* and opaque C++ type in the example).

Reply:
Done